lib/product-import.coffee

```diff
@@ -210,8 +210,11 @@ class ProductImport
       when 200 then @_summary.updated++
 
   _createProductFetchBySkuQueryPredicate: (skus) ->
-    skuString = "sku in (#{skus.map((val) -> JSON.stringify(val))})"
-    return "masterVariant(#{skuString}) or variants(#{skuString})"
+    if skus.length is 0
+      'masterVariant(sku in ("")) or variants(sku in (""))'
```

Comment:
Hello,

Thanks for your review I like your line of thought here (we can save the redundant API call)

I will like to propose we fix this in another issue, majorly because the product import and price import in impex are still currently broken without this fix

Danke